vector/src/main/java/im/vector/app/features/home/HomeDetailFragment.kt

```diff
@@ -422,8 +421,7 @@ class HomeDetailFragment @Inject constructor(
                 R.id.bottom_action_notification -> HomeTab.RoomList(RoomListDisplayMode.NOTIFICATIONS)
                 else                            -> HomeTab.DialPad
             }
-            // Tchap: Send event to shared VM to catch it in sub-fragments
-            sharedActionViewModel.post(HomeActivitySharedAction.SelectTab(tab))
```

Comment:
Check if we need to keep this VM `sharedActionViewModel` and the action `HomeActivitySharedAction.SelectTab`

Reply:
i removed HomeActivitySharedAction.SelectTab